neurolib/models/multimodel/model.py

```diff
@@ -236,44 +228,10 @@ def integrateChunkwise(self, chunksize, bold, append_outputs):
 
     def storeOutputsAndStates(self, results, append):
         # save time array
-        self.setOutput("t", results.time.values + self.start_t, append=append, removeICs=False)
-        self.setStateVariables("t", results.time.values + self.start_t)
+        self.setOutput("t", results.time.values, append=append, removeICs=False)
+        self.setStateVariables("t", results.time.values)
```

Comment:
This changes the time axis.

Reply:
We previously had different time axis behaviours for model and multimodel:
- for regular models, the time axis always started at 0 because it was reset for every chunk unless it was appended
- for multimodels, the time axis started at `self.start_t` from continued runs

We adapted the multimodel behaviour with a more elegant implementation without `self.start_t`.